client/src/lib/firebase/firestore-collections.ts

```diff
@@ -1,16 +1,21 @@
-import type { Lobby, PrivatePlayer, PromptAnswer } from "./firestore-types/lobby";
+import type { Lobby, PrivatePlayer, PromptAnswer, ChatRoom, ChatMessage } from "./firestore-types/lobby";
 import type { UserData } from "./firestore-types/users";
 import { collection, type CollectionReference, type DocumentReference } from "firebase/firestore";
 import { db } from "$lib/firebase/app";
 
+export const lobbyCollection = collection(db, "lobbies") as CollectionReference<Lobby>;
```

Comment:
I think it made sense for the userCollection to be above the lobbyCollection so thet all the lobby stuff was together.